pyanaconda/modules/storage/bootloader/base.py

```diff
@@ -902,6 +902,9 @@ def boot_prefix(self):
     def _set_console(self):
         """ Set console options based on boot arguments. """
         console = kernel_arguments.get("console", "")
+        if console is None:
```

Comment:
We should do that for an empty string as well. Therefore, the code can be simplified into:

```
console = kernel_arguments.get("console")
if not console:
    return
```

Reply:
OK and what about the log message. Seems that you want to drop it.

Reply:
I would suggest to log only the successful path with a message "The console is set to something with options something". It will be more informative anyway.